Include/cpython/object.h

```diff
@@ -324,18 +324,20 @@ PyAPI_FUNC(PyObject *) _PyObject_FunctionStr(PyObject *);
  * Py_DECREF.
  */
 
-#define Py_SETREF(op, op2)                      \
-    do {                                        \
-        PyObject *_py_tmp = _PyObject_CAST(op); \
-        (op) = (op2);                           \
-        Py_DECREF(_py_tmp);                     \
+#define Py_SETREF(op, op2)                                    \
```

Comment:
I dislike the original naming. `op` and `op2` does not carry any semantic value. Perhaps this is a good opportunity to change them to something more meaningful? `dst` and `src`? `to` and `from`?

Ditto for Py_XSETREF.

Reply:
Me too. I renamed arguments to dst and src. I also renamed temporary variables.

Reply:
Thanks! IMO a great improvement 😃